client/src/pages/RegisterLogin.css

```diff
@@ -25,31 +24,62 @@ h2 {
 	align-items: center;
 }
 
-/* Tablet/Desktop view */
-@media (min-width: 768px) {
+#messengerIcon {
```

Comment:
on in addition to utilizing makeStyles, you use both a stylesheet and customized styles - try to stay consistent between these two uses

Reply:
Is it okay to use embedded styles if it's just a small change? Like changing a margin or something or would it just be better to put it in a stylesheet?

Reply:
yeah i personally think its okay if it's just a small change! in that scenario, i wouldn't even bother putting it in an object variable, i would just apply it directly to the component